src/igraph/drawing/__init__.py

```diff
@@ -264,13 +264,16 @@ def plot(obj, target=None, bbox=(0, 0, 600, 600), *args, **kwds):
             warn("%s does not support plotting" % (obj,))
             return
         else:
-            plotter(
+            result = plotter(
                 backend,
                 target,
                 palette=palette,
                 *args,
                 **kwds,
             )
+            # NOTE: for matplotlib, result is the container Artist. It would be
+            # good to return this instead of target, like we do for Cairo.
+            # However, that breaks API so let's wait for a major release
```

Comment:
Open an issue and target it at 0.11, otherwise this will be forgotten.

Reply:
yep, done thank you